app/src/main/scala/org/alephium/explorer/persistence/queries/BlockDepQueries.scala

```diff
@@ -40,28 +40,29 @@ object BlockDepQueries {
       DBIOAction.successful(0)
     } else {
       //generate '?' placeholders for the parameterised SQL query
-      val placeholder = paramPlaceholder(rows = deps.size, columns = 3)
-      val query =
-        s"""
+      QuerySplitter.splitUpdates(rows = deps, columnsPerRow = 3) { (deps, placeholder) =>
```

Comment:
`QuerySplitter.splitUpdates` already checks for empty inputs [here](https://github.com/alephium/explorer-backend/blob/master/app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala#L123). The [test-case](https://github.com/alephium/explorer-backend/blob/master/app/src/test/scala/org/alephium/explorer/persistence/queries/QuerySplitterSpec.scala#L66). So the check above for `if(deps.isEmpty)` is probably not needed?

Reply:
good point, thx :+1: it's fixed